tests/conftest.py

```diff
@@ -3,6 +3,7 @@
 import pytest
 import requests
 import subprocess
+import sirepo.const
```

Comment:
don't import at this level in conftest, only at the place you need it.

Reply:
[308777d](https://github.com/radiasoft/sirepo/pull/4846/commits/308777d2447203787dc3df0950124714dd1d90f7)